v1.6.2/longhorn-engine/rockcraft.yaml

```diff
@@ -96,16 +96,6 @@ parts:
 
       ldconfig
 
-      # NOTE(aznashwan): pull and install liblonghorn
-      # https://github.com/longhorn/longhorn-engine/blob/v1.6.2/package/Dockerfile#L12-L19
-      LIBLONGHORN_COMMIT_ID="53d1c063b95efc8d949b095bd4bf04637230265f"
-      cd /usr/src
-      git clone https://github.com/rancher/liblonghorn.git
-      cd liblonghorn
-      git checkout ${LIBLONGHORN_COMMIT_ID}
-      make
-      make install
-
```

Comment:
Is this not used at all or included through another build step/part?

Reply:
We already have a ``build-liblonghorn`` step below, which clones this repo at this commit hash and runs ``make`` and ``make install``.